tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -146,11 +146,35 @@ export async function assertVc(context: IntegrationTestContext, subject: CorePri
 
     // step 7
     // check runtime version is present
-    assert.deepEqual(
-        vcPayloadJson.issuer.runtimeVersion,
-        { parachain: 9195, sidechain: 109 },
-        'Check VC runtime version: it should equal the current defined versions'
+    const specPaths = ['../../../runtime/litentry/src/lib.rs', '../../app-libs/sgx-runtime/src/lib.rs'].map(
+        (p) => path.resolve(process.cwd(), p) // resolve relative to working directory: integration-tests
     );
+    try {
+        const [nodeVersion, sidechainVersion] = await Promise.all(
+            specPaths.map((spec) => readFile(spec, { encoding: 'utf-8' }))
+        ).then((specs) =>
+            specs.map((spec) => {
+                const version = (spec.match(/spec_version:\s*(\d+)/) || [])[1];
```

Comment:
a bit flaky, but wanted to give it a try before moving the spec versions into json files.

Reply:
That's fine : )